openprescribing/templates/analyse.html

```diff
@@ -16,7 +16,7 @@
 
 <h1>Search GP prescribing data</h1>
 
-<p>Search 700 million rows of prescribing data, and get prescribing information by CCG or practice. You can search for any numerator over any denominator.</p>
+<p>Search 700 million rows of prescribing data, and get prescribing information by practice, CCG, STP or region. You can search for any numerator over any denominator.</p>
```

Comment:
"region" or "regional team" or "Regional Team"?

Reply:
"regional team" would be the most consistent, although I felt like the list was already getting a bit long winded so I went for something shorter. Can change if it bothers you though.